tasks/process_agent.py

```diff
@@ -95,3 +97,42 @@ def build(ctx, race=False, go_version=None, incremental_build=False, puppy=False
     }
 
     ctx.run(cmd.format(**args), env=env)
+
+
+@task
+def build_dev_image(ctx, image=None, push=False, base_image="datadog/agent:latest"):
+    """
+    Build a dev image of the process-agent based off an existing datadog-agent image
+
+    image: the image name used to tag the image
+    push: if true, run a docker push on the image
+    base_image: base the docker image off this already build image (default: datadog/agent:latest)
+    """
+    if image == None:
+        raise Exit(message="image was not specified")
+
+    with TempDir() as docker_context:
+        ctx.run("cp tools/ebpf/Dockerfiles/Dockerfile-process-agent-dev {to}".format(
+            to=docker_context + "/Dockerfile"))
+
+        ctx.run("cp bin/process-agent/process-agent {to}".format(
+            to=docker_context + "/process-agent"))
+
+        ctx.run("cp bin/system-probe/system-probe {to}".format(
+            to=docker_context + "/system-probe"))
+
+        with ctx.cd(docker_context):
+            ctx.run("docker build --tag {image} --build-arg AGENT_BASE={base_image} .".format(image=image, base_image=base_image))
+
+    if push:
+        ctx.run("docker push {image}".format(image=image))
```

Comment:
This seems potentially dangerous (eg: someone passes `datadog/agent:latest` thinking this parameter is the base image). Is it really needed?

Also, since there is no default for the `image` param, it could happen that different people push images with different names or naming conventions for the tags. What about providing a default for `image` (eg: datadog/process-agent-dev:$timestamp`) instead of leaving it as a free-form param?